modules/RTC/RTCUtils.js

```diff
@@ -1287,167 +1287,142 @@ class RTCUtils extends Listenable {
      * @returns {*} Promise object that will receive the new JitsiTracks
      */
     obtainAudioAndVideoPermissions(options = {}) {
-        const self = this;
-
         const dsOptions = {
             ...options.desktopSharingExtensionExternalInstallation,
             desktopSharingSources: options.desktopSharingSources
         };
 
-        return new Promise((resolve, reject) => {
-            const successCallback = function(stream) {
-                resolve(handleLocalStream(stream, options.resolution));
-            };
+        options.devices = options.devices || [ 'audio', 'video' ];
+        options.resolution = options.resolution || '720';
```

Comment:
maybe work for a subsequent commit, but these defaults feel like they should be coming from config or something.  or at least constants?

Reply:
hmm haven't we decided to not have any default options defined in the config, but have default handled by the code ?

Reply:
ok so not config but a constant then